config/application.rb

```diff
@@ -60,6 +60,7 @@ class Application < Rails::Application
     # Settings for external services.
     config.cache_enabled = ENV.fetch('CACHE_ENABLED', 'false').casecmp?('true')
     config.cache_expires_in_minutes = ENV.fetch('CACHE_EXPIRES_IN_MINUTES', '10').to_i
+    config.cache_expires_in_seconds = ENV.fetch('CACHE_EXPIRES_IN_SECONDS', '60').to_i
```

Comment:
Why do we need one more in seconds? In minutes could be 1, or if we go with seconds, we can adjust all the other code to use that, 10 minutes would be 600 secs.

The issue is that the more env variables we add, the more complex the deployment to maintain. We want to reduce variability.